cobigen-angular-client-generation/index.asciidoc

```diff
@@ -0,0 +1,93 @@
+= Generate Angular Client with devonfw
```

Comment:
You can now add a subtitle with the == syntax. See https://github.com/devonfw-tutorials/tutorials/wiki/Development